query-service-impl/src/main/java/org/hypertrace/core/query/service/postgres/PostgresBasedRequestHandler.java

```diff
@@ -64,8 +64,14 @@ public class PostgresBasedRequestHandler implements RequestHandler {
   private static final int DEFAULT_SLOW_QUERY_THRESHOLD_MS = 3000;
   private static final Set<Operator> GTE_OPERATORS = Set.of(Operator.GE, Operator.GT, Operator.EQ);
 
-  private static final Value NULL_VALUE =
+  private static final Value NULL_STRING_VALUE =
       Value.newBuilder().setValueType(ValueType.STRING).setString("null").build();
+  private static final Value NULL_INT_VALUE =
+      Value.newBuilder().setValueType(ValueType.STRING).setString("0").build();
```

Comment:
The value type appears off here, but I'm guessing this is the equivalent of what happens in pinot? If that's the case, can we leave a comment explaining because without context it looks like a bug.

Reply:
yes, currently, from query-service side, we are always returning string value, and conversation of an appropriate attribute.kind is handled at the gateway layer. we can add comments here.